assets/sass/components/_error_summary.scss

```diff
@@ -0,0 +1,27 @@
+$aplus-danger: #A50000;
+$aplus-background: #FFF;
+$width: 100%;
+$padding-amount: 5px;  
+$margin: 20px;
+
+
+.error-summary {
+	> .background {
```

Comment:
These are pretty short selectors. It's very possible that there is css class that targets `.background`. So maybe `&__background` (i.e. `.error-summary__background`).